### PR TITLE
Arrays hometask done.

### PR DESCRIPTION
**Pull request by kusem**

### CODE REVIEW

File: src/arrays/Arrays.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+class Arrays implements ArraysInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $new_array = [];
+        if (count($input) != 0) {
+            foreach ($input as $currentElement) {
```
Comment (yderevianko-cartera) [inline src/arrays/Arrays.php]: please use php array function

File: src/arrays/Arrays.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+namespace arrays;
+
+class Arrays implements ArraysInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $new_array = [];
```
Comment (yderevianko-cartera) [inline src/arrays/Arrays.php]: please use camelCase names

File: src/arrays/Arrays.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+     */
+    public function getUniqueValue(array $input): int
+    {
+        $uniqueArray = array_unique($input, SORT_NUMERIC);   //getting single values
+        $count_values = array_count_values($input);
+
+        foreach ($uniqueArray as $value) {                        //converting single values array into unique values
+            if ($count_values[$value] > 1) {
+                $key = array_search($value, $uniqueArray);
+                unset($uniqueArray[$key]);
```
Comment (yderevianko-cartera) [inline src/arrays/Arrays.php]: it is not good to unset array that you are looping

File: src/arrays/Arrays.php [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        $tagsArray = [];
+
+        foreach ($input as $currentElement) {
+            foreach ($currentElement['tags'] as $currentElementTag) {
+                $tagsArray[$currentElementTag][] = $currentElement['name'];
+            }
+        }
+
+        foreach ($tagsArray as $currentTagName => $currentTagContent) {
+            sort($currentTagContent);
```
Comment (yderevianko-cartera) [inline src/arrays/Arrays.php]: it is not a good idea to sort inside foreach loop

**Review by yderevianko-cartera — CHANGES_REQUESTED** [review]

File: src/arrays/Arrays.php (hunk 1 above)
Comment (kusem) [reply src/arrays/Arrays.php]: > please use php array function

do you mean I'd replace for/foreach with array_walk_recursive/array_fill ?

Comment (yderevianko-cartera) [reply src/arrays/Arrays.php]: yes, it is just for learning about php array functions and how to use them.
Main point it still should work and tests should pass.

File: src/arrays/Arrays.php (hunk 4 above)
Comment (kusem) [reply src/arrays/Arrays.php]: All changes done. Now I'm proud of my code here ;)

File: src/arrays/Arrays.php (hunk 3 above)
Comment (kusem) [reply src/arrays/Arrays.php]: > it is not good to unset array that you are looping

All changes done. Now I'm proud of my code here ;)

File: src/arrays/Arrays.php (hunk 2 above)
Comment (kusem) [reply src/arrays/Arrays.php]: > please use camelCase names

All changes done. Now I'm proud of my code here ;)

File: src/arrays/Arrays.php (hunk 1 above)
Comment (kusem) [reply src/arrays/Arrays.php]: > yes, it is just for learning about php array functions and how to use them. Main point it still should work and tests should pass.

All changes done. Now I'm proud of my code here ;)

**Review by yderevianko-cartera — APPROVED** [review]

**Pull request merged**